packages/engine/src/scene/components/ParticleSystemComponent.ts

```diff
@@ -886,7 +889,7 @@ export const ParticleSystemComponent = defineComponent({
     const metadata = useHookstate({ textures: {}, geometries: {}, materials: {} } as ParticleSystemMetadata)
     const sceneID = useOptionalComponent(entity, SourceComponent)?.value
     const rootEntity = useHookstate(getMutableState(GLTFSourceState))[sceneID ?? ''].value
-    const sceneLoaded = GLTFComponent.useSceneLoaded(rootEntity)
+    const sceneLoaded = hasComponent(rootEntity, GLTFComponent) ? GLTFComponent.useSceneLoaded(rootEntity) : true
```

Comment:
hooks cannot be conditional. These logic needs to be handled with either a subreactor or something other way